lib/node_modules/@stdlib/blas/ext/base/dfill/docs/repl.txt

```diff
@@ -28,7 +28,7 @@
     Returns
     -------
     x: Float64Array
-        Input array `x`.
+        Output array.
```

Comment:
```suggestion
        Input array.
```

We can actually leave as is, for this package, as the input array is what is returned. In other packages, we support providing an output array, but the intent of this package is to mutate the input array.